code/src/components/TodoAmount.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import styled from 'styled-components/macro';
+
+const TaskInfo = styled.div`
+  display: flex;
+  justify-content: space-around;
+  height: 3vh;
+  &.no-tasks {
+    display: none;
+  }
+
+  .task-info-text {
+    font-weight: 400;
+    margin: 0;
+  }
+`;
+
+const TodoAmount = () => {
+  const todoListItems = useSelector((store) => store.todos.todoListItems);
+  const completedListItems = todoListItems.filter(
+    (listItem) => listItem.isComplete
+  );
+  const uncompletedListItems = todoListItems.filter(
+    (listItem) => !listItem.isComplete
+  );
+
+  return (
+    <TaskInfo
+      className={
```

Comment:
Really nice logic to style when you have no tasks! I just had it read "todo:0, finished:0" but this is so much cleaner!